bite-size-notes/week-8-e-Bayes Rule-RBE-RBF.md

```diff
@@ -0,0 +1,94 @@
+# Bayes Rule, Recursive Bayesian Estimator, and the Recursive Bayesian Filter
+
+## Bayes's Rule
+
```

Comment:
This review of Bayes' rule is very comprehensive, and works great as an introduction to the topic!